orbit/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## Orbit 1.32.0 (Aug 29, 2024)
+
+* Bumped macadmins extension to use SOFA feed sofafeed.macadmins.io
+
+* In Fleet Desktop, "My Device" status is now refreshed when user clicks on "My Device" or "Self-service" dropdown option.
```

Comment:
```suggestion
* Fixed an issue where In Fleet Desktop, "My Device" status is now refreshed when user clicks on "My Device" or "Self-service" dropdown option.
```

Reply:
Mhm the line is explaining the fix not the issue.

Reply:
Has to be past tense

Reply:
Ok, how about:
```
* Fixed Fleet Desktop to refresh host status when the user clicks on "My Device" or "Self-service" dropdown option.
```

Reply:
Great!

Reply:
Edited the comment.